sda/internal/schema/schema_test.go

```diff
@@ -193,7 +193,7 @@ func TestValidateJSONIngestionAccessionRequest(t *testing.T) {
 		User:     "JohnDoe",
 		FilePath: "path/to file",
 		DecryptedChecksums: []Checksums{
-			{Type: "sha256", Value: "da886a89637d125ef9f15f6d676357f3a9e5e10306929f0bad246375af89c2e2"},
+			{Type: "sha256", Value: "da886a89637d125ef9f15f6d676357f3a9e5e10306929f0bad246375afc2e2"},
```

Comment:
Why changing only 2 out of 11?

Reply:
that test expects to fail as it misses md5, at least that is how i read it, personally i would remove it.
so the other direction i took is to fail the sha256 value

Reply:
i changed that to be md5 instead of sha256 but i am fine with any other suggestion